subsys/ipc/ipc_service/lib/Kconfig

```diff
@@ -28,3 +28,19 @@ menuconfig IPC_SERVICE_ICMSG
 if IPC_SERVICE_ICMSG
 	rsource "Kconfig.icmsg"
 endif
+
+config IPC_SERVICE_ICMSG_ME
+	bool "icmsg IPC library with multi-endpoint functionality"
+	select IPC_SERVICE_ICMSG
+	select EVENTS
+	help
+	  Multi-endpoint functionality for the icmsg library
+
+config IPC_SERVICE_ICMSG_ME_NOCOPY_RX
+	bool
```

Comment:
is this an hidden option?

Reply:
It was my intention to make this hidden. It is selected by backends when needed.